app/src/main/java/app/revanced/manager/ui/viewmodel/ImportExportViewModel.kt

```diff
@@ -34,6 +34,45 @@ import java.nio.file.StandardCopyOption
 import kotlin.io.path.deleteExisting
 import kotlin.io.path.inputStream
 
+sealed class ResetDialogState(
+    val title: String,
+    val description: String,
+    val onConfirm: () -> Unit
+) {
+    object None : ResetDialogState("", "", { })
+
+    class Keystore(onConfirm: () -> Unit) : ResetDialogState(
+        title = "Keystore regeneration",
+        description = "You are about to regenerate your keystore the manager will use during the patching process.\n\n" +
+                "You will not be able to update the previously installed apps from this source.",
```

Comment:
These strings are not translatable. Use string resources instead. One way to implement this would be to change `title` and `description` into string resource ids and store the extra parameters (package name, bundle name) in an array. Then you would be able to do something like this in the UI: `stringResource(resetDialogState.description, *resetDialogState.descriptionParams)`